sdk/rust/src/ethereum.rs

```diff
@@ -0,0 +1,10 @@
+use crate::dstack_client::GetKeyResponse;
+use ethers::signers::LocalWallet;
```

Comment:
`LocalWallet` has changed to https://docs.rs/alloy/latest/alloy/signers/local/index.html

Reply:
Fixed